### PR TITLE
Fix some typos in test/e2e/apps/statefulset.go

### PR DESCRIPTION
**Pull request by billy2180**
1. fix some typos
2. fix deprecated method Core() to CoreV1()

```release-note
NONE
```

### CODE REVIEW

File: test/e2e/apps/statefulset.go [hunk 1]
```diff
@@ -188,7 +188,7 @@ var _ = SIGDescribe("StatefulSet", func() {
 			)).To(Succeed(), "wait for pod %q to be released", pod.Name)
 
 			// If we don't do this, the test leaks the Pod and PVC.
-			By("Readding labels to the stateful set's pod")
```
Comment (xiangpengzhao) [inline test/e2e/apps/statefulset.go]: As per the code below, this seems like not a typo. It meant to say `re-add`, IMO.

Comment (billy2180) [inline test/e2e/apps/statefulset.go]: ok,i revert it

**Comment by soltysh** [comment]
@billy2180 please squash your changes and I'll approve it

**Comment by billy2180** [comment]
@soltysh Done, PTAL

**Pull request closed without merging**